src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = WhatIsSex(people, 'm');
```

Comment:
Змінні починаємо з великої буквою для:

1.  функцій-конструкторів
2. класів

у тебе звичайна функцію
```suggestion
  const man = whatIsSex(people, 'm');
```

Reply:
thanks for the explanation, fixed it